lib/traject/macros/canvas.rb

```diff
@@ -27,6 +27,16 @@ def extract_canvas_label
       end
     end
 
+    ##
+    # Note: This method assumes an "enhanced" canvas with additional properties
+    # added beyond the IIIF Canvas model.
+    def extract_canvas_label_sort
+      lambda do |record, accumulator, _context|
+        labels = [record['label'].to_s, record['manifest_label'].to_s].reject(&:empty?)
+        accumulator << labels.reverse.join(': ')
```

Comment:
Any particular reason we're reversing here and not ordering the array above?  I know it's the same array as in the other method, but just wondering if we can do w/o the reverse. /shrug

Reply:
No reason. I changed it to order properly. Thanks for the suggestion.